projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -130,6 +130,15 @@ function wpcom_launchpad_get_task_definitions() {
 				return '/me/account';
 			},
 		),
+		'verify_domain_email'             => array(
```

Comment:
Having gone through the ICANN discussion, it doesn't seem like this definition is precise, right? Are we dispatching the ICANN verification e-mail when this is clicked?

Reply:
It will be precise once I update `is_domain_email_unverified` which I am doing now.